chainer/function_hook.py

```diff
@@ -118,7 +136,8 @@ def deleted(self, function=None):
 
         Args:
             function(~chainer.FunctionNode): Function object to which
-                the function hook is deleted.
+                the function hook is deleted. ``None`` if the function hook
+                had been registered globally.
```

Comment:
has been?

Reply:
I'm not confident about my expression either, but the hook is unregistered (the state of "being registered" is negative) at the moment of this callback.

Reply:
According to @hvy, "had been" is okay but "was" is more natural expression. I fixed as such.